webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -76,10 +74,6 @@ export const loadSubscriptions = (extendedParams = {}) => async (dispatch) => {
       response: data,
       search: extendedParams.search,
     });
-    const poolIds = filterRHSubscriptions(data.results).map(subs => subs.id);
-    if (poolIds.length > 0) {
-      dispatch(loadAvailableQuantities({ poolIds }));
```

Comment:
I wanted to avoid making this API request if we already knew that making a connection to upstream RHSM would not be successful. Rather than hacking this method (my first approach) I decided to decouple `loadAvailableQuantities` and call it directly from `SubscriptionsPage`.